backend/postgres/postgres.go

```diff
@@ -6,6 +6,10 @@ package postgres
 import (
 	"context"
 	"fmt"
+	"github.com/opentracing/opentracing-go"
```

Comment:
Please order imports accordingly

Reply:
Sure. See following commit with `$ gofumpt -w .` having been executed on the whole project.